README.md

```diff
@@ -56,6 +56,31 @@ $ docker run -d --link selenium-hub:hub selenium/node-chrome:2.53.1
 $ docker run -d --link selenium-hub:hub selenium/node-firefox:2.53.1
 ```
 
+### Specifying versions
+
+By default the latest stable versions will be used, however you can override
+these using the following environment variables:
+
+- `SELENIUM_VERSION` - version of the Selenium server
+  - latest stable: `2.53.1`
+  - latest beta: `3.0.0-beta3`
+- `FIREFOX_VERSION` - version of Firefox (only applies to Firefox images)
+  - latest release: `49.0.1`
+  - previous release: `48.0.2`
+  - latest ESR: `45.4.0esr`
+  - previous ESR: `38.8.0esr`
+
+Note that as the latest releases of Firefox work best with Selenium 3.0, which
+is currently in beta, the latest release supported by 2.53.x is `47.0.1` and is
+also available (and the default).
+
```

Comment:
If I remember my test before, the 2.53.x can work with Firefox 47.0.1 if not use caps["marionette"] = True.

if you use caps["marionette"] = True to run selenium test. it would fail. 

This is my only concern here for various versions matrix.

Other code LGTM.